modules/text-input/react/spec/TextInput.spec.tsx

```diff
@@ -1,50 +1,72 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
-import ReactDOMServer from 'react-dom/server';
+import {render, fireEvent} from '@testing-library/react';
 import TextInput from '../lib/TextInput';
+import ReactDOMServer from 'react-dom/server';
 import {axe} from 'jest-axe';
 import FormField from '@workday/canvas-kit-react-form-field';
 
-describe('TextInput', () => {
-  test('TextInput should spread extra props', () => {
-    const component = mount(<TextInput data-propspread="test" onChange={jest.fn()} />);
-    const input = component
-      .find('input') // TODO: Standardize on prop spread location (see #150)
-      .getDOMNode();
-    expect(input.getAttribute('data-propspread')).toBe('test');
-    component.unmount();
+const id = 'Test Text Input';
+const label = 'Test Text Input';
+const placeholder = 'Test Text Input';
+const value = 'Test Text Input';
+
+describe('Text Input', () => {
+  const cb = jest.fn();
+  afterEach(() => {
+    cb.mockReset();
   });
-});
 
-describe('Text Input Accessibility', () => {
-  test('Text Input in a FormField should pass axe DOM accessibility guidelines', async () => {
-    const html = ReactDOMServer.renderToString(
-      <FormField
-        label="My Field"
-        inputId="my-input-field"
-        hintText="Helpful text to resolve error"
-        hintId="my-input-field-error"
-        error={FormField.ErrorType.Error}
-      >
-        <TextInput placeholder="Placeholder" value={'Hello'} onChange={jest.fn()} />;
-      </FormField>
-    );
-    expect(await axe(html)).toHaveNoViolations();
+  describe('when rendered', () => {
+    describe('with an placeholder', () => {
+      it('should render a text input with placeholder', () => {
+        const {getByPlaceholderText} = render(
```

Comment:
I think getting by the placeholder text and then asserting it has a placeholder probably won't work here. You'll never get a proper failure since you wouldn't be able to assert on anything if the placeholder fails.

Maybe a `getByTestId` instead?